src/kiebot-kogito-cr.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: app.kiegroup.org/v1beta1
+kind: KogitoBuild
+metadata:
+  name: kiebot
+spec:
+  type: RemoteSource
+  #env:
+  # env can be used to set variables during build
+  #- name: MY_CUSTOM_ENV
+  #  value: "my value"
+  gitSource:
+    uri: 'https://github.com/kiegroup/kiebot'
+  runtimeImage: quay.io/kiegroup/kogito-runtime-jvm:1.35
+---
+apiVersion: app.kiegroup.org/v1beta1
+kind: KogitoRuntime
+metadata:
+  name: kiebot
+spec:
+  replicas: 1
+  env:
+    - name: QUARKUS_GITHUB_APP_APP_ID
+      valueFrom:
+        secretKeyRef:
+          key: GITHUB_APP_ID
+          name: kiebot-secret
+    - name: QUARKUS_GITHUB_APP_APP_NAME
+      valueFrom:
+        secretKeyRef:
+          key: GITHUB_APP_NAME
+          name: kiebot-secret
+    - name: QUARKUS_GITHUB_APP_WEBHOOK_PROXY_URL
```

Comment:
I am not sure we need that one. 
See https://github.com/quarkiverse/quarkus-github-app/blob/main/runtime/src/main/java/io/quarkiverse/githubapp/runtime/config/GitHubAppRuntimeConfig.java#L57

Reply:
maybe we can keep, for example purpose only.

Reply:
nope, because it breaks the launch of the application ...

Reply:
I don't get what you mean ?

Reply:
keep this file as example only, but when deploying, you edit it to your needs.

Reply:
Well, this file would be used only for prod or ?
for local dev, it is all explained in docs: https://quarkiverse.github.io/quarkiverse-docs/quarkus-github-app/dev/create-github-app.html
I don't see the point of keeping this here